autofit/non_linear/abstract_search.py

```diff
@@ -151,6 +153,81 @@ def __init__(
 
     __identifier_fields__ = tuple()
 
+    def optimise(

```

Comment:
Actually moved the important stuff here - this way _any_ autofit search can now be used in conjunction with graphical